JSONFormat.go

```diff
@@ -39,7 +45,7 @@ func (j jsonEntry) MarshalJSON() ([]byte, error) {
 		return nil, ErrNoMessage
 	}
 
-	_, err := buf.WriteString(fmt.Sprintf("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":\"%s\"", j.levelKey, j.Level, j.eventKey, j.Event, j.messageKey, j.Message))
+	_, err := buf.WriteString(fmt.Sprintf("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":\"%s\"", j.levelKey, j.Level, j.eventKey, j.Event, j.messageKey, escapeSpecialChars(j.Message)))
```

Comment:
I don't understand.. this is the normal way to do json serialization.  Since there isn't a standard way to do what we wanted to do (a flat map, if I recall correctly), I created a custom method. No magic.
I think this could've been done better in this way:
`_, err := buf.WriteString(fmt.Sprintf("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":%s", j.levelKey, j.Level, j.eventKey, j.Event, j.messageKey, strconv.Quote(j.Message)))`
But beyond that.. why would you want to do anything else?